client/admin/users/EditUser.js

```diff
@@ -102,7 +102,7 @@ export function EditUser({ data, roles, ...props }) {
 
 	const canSaveOrReset = hasUnsavedChanges || avatarObj;
 
-	const prepend = useMemo(() => <UserAvatarEditor username={data.username} setAvatarObj={setAvatarObj}/>, [data.username]);
+	const prepend = useMemo(() => <UserAvatarEditor userId={data._id} username={data.username} setAvatarObj={setAvatarObj}/>, [data._id, data.username]);
```

Comment:
I thought `userId` was enough to get the avatar URL **and** the username. Is this line passing the `username` prop to keep `UserAvatarEditor` compatible with something else?

Reply:
Yes, I actually need the current username in order to render the avatar reset button.

Reply:
Just to be clearer, if I change the username, I wan't the default avatar to update before I save the data.